### ISSUE TITLE
Remove the posts table that has been left in the site's design (finish migration to div's instead)

### ISSUE THREAD

**Issue by levans88** [levans88/ProjectKB#18]
I should be using div's instead of a table and using CSS to align things.  Tables break screen readers and cause other problems.

I mostly removed the original table but it is still "wedged" into the design and styling.  This is causing a display width issue in some browsers.
